### PR TITLE
feat: add test for explain

### PR DESCRIPTION
**Pull request by kubo-hiroto**
- Fixed an issue where removal of symbols in column names causes KeyError.
- This PR is related to https://github.com/sapientml/core/pull/14
- The test_explaint is not enabled because it must be imported at the same time as the PR above or an error will occur.

### CODE REVIEW

File: sapientml/params.py [hunk 1]
```diff
@@ -379,7 +379,6 @@ def __init__(
             filename = "training." + ("pkl" if save_datasets_format == "pickle" else "csv")
             self.training_data_path = str(self.output_dir / filename)
             save_file(self.training_dataframe, self.training_data_path, csv_encoding, csv_delimiter)
-            self.training_data_path = str(Path(self.training_data_path).relative_to(self.output_dir))
```
Comment (AkiraUra) [inline sapientml/params.py:382]: Why did you remove this line?

Comment (kubo-hiroto) [reply sapientml/params.py:382]: In line 380, "self.training_data_path" is the absolute path, but in line 382, it is changed to only the file name, and failed to load the file when reloading.

Comment (AkiraUra) [reply sapientml/params.py:382]: Thank you. No problem.

**Review by AkiraUra — APPROVED** [review]

**Review by kimusaku — APPROVED** [review]
